src/test/java/org/itsallcode/openfasttrace/report/view/html/TestHtmlReportDetails.java

```diff
@@ -0,0 +1,48 @@
+package org.itsallcode.openfasttrace.report.view.html;
+
+/*-
+ * #%L
+ * OpenFastTrace
+ * %%
+ * Copyright (C) 2016 - 2018 itsallcode.org
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */
+
+import org.itsallcode.openfasttrace.report.view.Viewable;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class TestHtmlReportDetails extends AbstractTestHtmlRenderer
+{
+    @Override
+    @BeforeEach
+    public void prepareEachTest()
```

Comment:
Is this necessary? The parent class method should be automatically called?

Reply:
That was done by automatic refactoring. Good that you saw it, I completely missed that part.